Main.java

```diff
@@ -9,7 +9,8 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // Prints Hellow World to the screen
+    System.out.println("Hello world");

```

Comment:
The reason it didn't pass the test is because you are missing a ! at the end of "Hello World"

Reply:
I think I got it all figured out!

Reply:
@MrLamont-HHSS I got it working now. Think you can have another look at it?